angularjs-1.5-basics/src/app/angular-1.5-basics-main.routing.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+angular.module('angular-1.5-basics-main')
```

Comment:
__[jshint]__
* 'angular' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->